lib/activerecord_slotted_counters/adapters/pg_sqlite_upsert.rb

```diff
@@ -2,15 +2,19 @@
 
 module ActiveRecordSlottedCounters
   module Adapters
-    class PgUpsert
+    class PgSqliteUpsert
```

Comment:
The adapter pattern means separating implementations, not mixing them up. Please, fix this. There should not be any changes to the existing PgUpsert class.

Reply:
yeah
I just thought that `SQLite` adapter has the same functionality, and maybe the `apply?` method could resolve it

but I got it, fixed it
and in that case, the duplication is not bad, because we have two different implementations

thanks